commit b93239da427a5019a9742d44e50e9996c1b04ea9
Author: Nathan Pratta Teodosio <nathan.teodosio@canonical.com>
Date:   Fri Jun 19 09:06:46 2026 -0700

    Only pass sysroot if use_sysroot.
    
    Fixes the build when use_sysroot = false as then 'sysroot' variable is empty.
    
    Bug: None
    Change-Id: I7fc7ba35f3ba416af0cbd8df94393dc901724b8b
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/7927226
    Reviewed-by: Sylvain Defresne <sdefresne@chromium.org>
    Commit-Queue: Andrew Grieve <agrieve@chromium.org>
    Reviewed-by: Andrew Grieve <agrieve@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1649706}

--- a/build/modules/BUILD.gn
+++ b/build/modules/BUILD.gn
@@ -218,7 +218,7 @@ if (use_unified_system_module) {
         # otherwise.
         "--append-module=std.new:export new_h\\nexport vcruntime_exception",
       ]
-    } else {
+    } else if (use_sysroot) {
       # We need to pass the sysroot in so that it can scan it to generate a
       # modulemap for the sysroot headers.
       args += [
